src/reference_implementations/prompting_vector_llms/llm_prompting_examples/llm_prompt_boolq.ipynb

```diff
@@ -222,279 +312,677 @@
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### N=1 Few-Shot Examples\n",
     "\n",
-    "In this section, we use a single fixed example in our prompt before asking our model to answer the question that we care about for each data point. In the original [GPT-3 paper](https://arxiv.org/pdf/2005.14165.pdf), this resulted in a large jump in performance on the BoolQ task, which we also see here."
+    "\n",
+    "### Zero-shot Prompting (Poor Label Space Choice)\n",
+    "\n",
+    "Because there are only two categories in our label space (yes/no) and the question is fairly \"leading.\" That is,  the model tends to answer in our label space through greedy decoding fairly well, (though there are still 11 instances where it doesn't). However, it's not always easy to select a label space that the model consistently produces an answer in for zero-shot (see [AG's News Notebook](src/reference_implementations/prompting_vector_llms/llm_prompting_examples/llm_prompt_ag_news.ipynb)). There are techniques that can be used to help this (like re-writing the prompt or using few-shot prompts), but what happens if we choose a bad label space?\n",
+    "\n",
+    "Let's try it out below. Instead of expecting a \"Yes\" or \"No\" answer, let's \"expect\" True or False."
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "label_map_1 = {0: \"False\", 1: \"True\"}\n",
+   "execution_count": 22,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Prompt Example\n",
+      "Shear wall -- In structural engineering, a shear wall is a structural system composed of braced panels (also known as shear panels) to counter the effects of lateral load acting on a structure. Wind and seismic loads are the most common building codes, including the International Building Code (where it is called a braced wall line) and Uniform Building Code, all exterior wall lines in wood or steel frame construction must be braced. Depending on the size of the building some interior walls must be braced as well.\n",
+      "question: is a shear wall a load bearing wall?\n",
+      "answer (True or False):\n"
+     ]
+    }
+   ],
+   "source": [
+    "label_map = {0: \"False\", 1: \"True\"}\n",
     "label_ordering = [\"False\", \"True\"]\n",
-    "demonstrations_1 = create_demonstrations(\n",
-    "    bool_q_train_titles, bool_q_train_passages, bool_q_train_questions, bool_q_train_labels, label_map_1, 1\n",
-    ")\n",
-    "prompts_1 = create_prompts(demonstrations_1, bool_q_test_titles, bool_q_test_passages, bool_q_test_questions)"
+    "lowercase_labels = [label.lower() for label in label_ordering]\n",
```

Comment:
Yeah I know what you mean. I'm careful about it to make sure it doesn't impact performance, but I can see how it might be confusing. However, in the name of time, I'm going to leave it for now and put a note to myself to address it later if there is time.